lib/api.v2.js

```diff
@@ -112,6 +114,14 @@ function _findOne(collection, query, exactMatch, cb) {
     model = _.first(libs);
   }
 
+  // move etag resource header into place
+  if (exactMatch) {
+    var etagsCollection = dbs[config.etagsCollection]
```

Comment:
@megawac sorry I just re-read this (thought it was '*were* you adding items', not 'where'); these are added [here](https://github.com/jsdelivr/api/blob/master/tasks/sync.js#L169), during the sync process.